sockjs-protocol-0.3.3.py

```diff
@@ -819,21 +819,36 @@ def test_request_headers_cors(self):
                 headers={'Origin': 'test', 'Access-Control-Request-Method': 'POST', 'Access-Control-Request-Headers': 'a, b, c'})
         self.assertTrue(r.status == 204 or r.status == 200)
         self.verify_cors(r, 'test')
-        self.assertEqual(r['Access-Control-Allow-Headers'], 'a, b, c')
+        # A preflight request can benefit from returning all headers that it allows a client to send, since these
+        # headers will be stored in the preflight cache (similar to how the request methods are stored) which can 
+        # potentially save a round trip for subsequent request that send other/more headers.
```

Comment:
I thought that all requests that the SockJS client did had cache busting query strings. Making it impossible to actually cache these responses.